packages/analyticsdx-template-lint/src/schemas/layout-schema.json

```diff
@@ -46,135 +53,105 @@
+          "navigation": {
+            "$ref": "#/definitions/navigation",
+            "description": "Configure the node in the navigation panel for this page."
+          },
+          "guidancePanel": {
+            "$ref": "#/definitions/guidancePanel",
+            "description": "The guidance panel specification for this page."
+          },
+          "layout": { "doNotSuggest": true },
+          "header": { "doNotSuggest": true },
+          "groups": { "doNotSuggest": true }
+        },
+        "oneOf": [
+          {
+            "required": ["type", "layout"],
             "properties": {
               "type": {
-                "type": "string",
-                "description": "Layout type",
-                "enum": ["SingleColumn", "TwoColumn"],
-                "enumDescriptions": [
-                  "A page layout with a single panel of items.",
-                  "A page layout with left and right panels of items."
-                ]
+                "const": "Configuration"
               },
-              "header": {
-                "type": ["null", "object"],
-                "additionalProperties": false,
-                "description": "Header text to display at the top of the page, under the title.",
-                "properties": {
-                  "text": {
-                    "type": ["null", "string"],
-                    "description": "Header text. This can contain {{...}} expressions.",
-                    "defaultSnippets": [{ "label": "\"\"", "body": "${0}" }]
-                  },
-                  "description": {
-                    "type": ["null", "string"],
-                    "description": "Header description text, displayed under the header text. This can contain {{...}} expressions.",
-                    "defaultSnippets": [{ "label": "\"\"", "body": "${0}" }]
-                  }
-                },
-                "defaultSnippets": [
-                  {
-                    "label": "New header",
-                    "body": {
-                      "text": "${1:Header text}",
-                      "description": "${2}"
-                    }
-                  }
-                ]
+              "groups": {
+                "doNotSuggest": true,
+                "errorMessage": "Property groups is not allowed.",
+                "not": true
               },
-              "center": {
-                "doNotSuggest": true
-              },
-              "right": {
-                "doNotSuggest": true
+              "header": {
+                "doNotSuggest": true,
+                "errorMessage": "Property header is not allowed.",
+                "not": true
               },
-              "left": {
-                "doNotSuggest": true
+              "layout": {
+                "$ref": "#/definitions/layout",
+                "description": "The layout specification for the page.",
+                "doNotSuggest": false
               }
-            },
```

Comment:
For readability, I moved the SingleColumn/TwoColumn stuff also down to the definitions, since it also has this weird discriminated-union `anyOf` logic.